app/resources/api/v1/protected/session_resource.rb

```diff
@@ -0,0 +1,28 @@
+class Api::V1::Protected::SessionResource < JSONAPI::Resource
+  immutable # no CUD through controller
```

Comment:
This Resource/DTO is marked as immutable.  It cannot be altered through the controller or by api calls.